log_test.go

```diff
@@ -0,0 +1,7 @@
+package dht
+
+import "log"
+
+func init() {
+	log.SetFlags(log.Flags() | log.Llongfile)
```

Comment:
Is this needed? We don't use std `log` by default.

Reply:
It only sets the flag when running tests, where I use it for some debugging. It's very difficult to isolate the source of debug messages with just Lshortfile in the libp2p projects (to mirror IPFS logging) due to the fragmentation of the repositories.